server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -1101,7 +1107,7 @@ public void run() {
       log.error("Error initializing metrics, metrics will not be emitted.", e1);
     }
 
-    recoveryManager = new RecoveryManager(this, TIME_TO_CACHE_RECOVERY_WAL_EXISTENCE);
+    recoveryManager = new RecoveryManager(this, timeToCacheRecoveryWalExistence);
```

Comment:
This should pass a LongSupplier to compute the value dynamically from the current value of the configurable interval time.

Reply:
I'm not sure that makes sense here. This is done during Manager startup before clients can connect to it. The RecoveryManager is created and the value of `timeToCacheRecoveryWalExistence` is used in the RecoveryManager constructor to set the expiration time on the Caffeine CacheBuilder. This is only called once, not in a loop, and I don't think the Caffeine cache can be updated after construction.